health-and-fitness-organizer/src/components/Navbar.module.css

```diff
@@ -1,7 +1,57 @@
+.coName {
+}
+
+.logo {
+  font-size: xxx-large;
+  margin-top: 2px;
+}
+
+.logoContainer {
+  /* font-size: xx-large; */
+}
+
+.logoAndName {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
 .navbar {
   display: flex;
-  flex-direction: row;
+  align-items: center;
+  gap: 100px;
+  background-color: black;
+  color: white;
+  height: 50px;
+  padding: 0 1% 0 1%;
```

Comment:
repeated padding and margin can be shorthanded.  the above can just be padding: 0 1% and it automatically knows that the bottom and left padding should be 0 1%

Reply:
fixed: used shorthand